### PR TITLE
Threefold links

### PR DESCRIPTION
**Pull request by gmachtel**
Fixes #

## Proposed Changes

  - Add ThreeFold repos
  -
  -

### CODE REVIEW

File: content/resources.json [hunk 1]
```diff
@@ -239,7 +239,76 @@
           "title": "🔷 Gaia-X Test Network Ocean Faucet",
           "desc": "The faucet to acquire your first Ocean test network tokens.",
           "url": "https://faucet.gaiaxtestnet.oceanprotocol.com/send?address=0x01e66950353400E93AEe7F041C0303103E2ef5Ab"
-        }
+        },
```
Comment (moritzkirstein) [inline content/resources.json:242]: The resource array from the previous category as well as the category then need to be closed here.
replace the `},` with
```
    }
  ]
},
```

File: content/resources.json [hunk 2]
```diff
@@ -239,7 +239,76 @@
...
+             "desc": "Inventory of Weblets, A weblet is a compiled javascript component which can be embedded in any website.",
+             "url": "https://github.com/threefoldtech/grid_weblets/tree/development/src/elements"
+           },
+           {
+             "title": "Threefold Presentation for Structura-X",
+             "desc": "Inventory of Weblets, A weblet is a compiled javascript component which can be embedded in any website.",
+             "url": "https://docs.google.com/presentation/d/1ms0WQZ_Uh1O748M2cWMCEq5xvOtX4j9BUkICmcrJuS0/edit#slide=id.g9621af315e_0_0"
+           }
+         ]
       ]
```
Comment (moritzkirstein) [inline content/resources.json:312]: remove `]`

**Review by moritzkirstein — CHANGES_REQUESTED** [review]
Thanks for the submission of those resources!
To successfully deploy the changes, please resolve JSON parsing errors.

@kaimeinke please have a look at where these resources fit best, and verify if a new category / track is the correct solution here.

**Review by gmachtel — COMMENTED** [review]
Now added as separate track, but idea was to have it as subtract of 6-MVG

**Comment by kaimeinke** [comment]
Dear @gmachtel , i fixed the errors and put the content changes to https://github.com/deltaDAO/mvg-hackathon-overview/pull/24.
Best Kai

**Pull request closed without merging**